mevislab.github.io/content/examples/data_objects/contours/example1/index.md

```diff
@@ -21,9 +26,18 @@ Contours (in MeVisLab CSO's) can be created by many modules, such as:
 * `SoCSOIsoEditor`
 * `SoCSOLiveWireEditor`
 
-Whenever creating contours, they are managed and temporarily stored by a `CSOManager`. In this example, contours are created and colors and styles of these CSOs are customized by using a `SoCSOVisualizationSettings` module.
+{{<alert class="info" caption="Info">}}
+Whenever Contour Segmented Objects are created, they are temporarily stored by and can be managed with the `CSOManager`. 
+{{</alert>}}
+
+In this example, contours are created and colors and styles of these CSOs are customized by using the `SoCSOVisualizationSettings` module.
 
 ![Screenshot](/examples/data_objects/contours/example1/image.png)
 
+## Summary
++ Contours are stored as their own abstract data type called Contour Segmented Objects (Often abbreviated to *CSO*).
```

Comment:
Full sentences in parentheses are begun with an upper case character, that's why I wrote "Often" instead of "often" -
one can argue whether or not "Often abbreviated to CSO" is a full sentence though 

https://www.crowdcontent.com/resources/writer/university/punctuation/parentheses-and-brackets/#:~:text=When%20a%20whole%20sentence%20is%20in%20parentheses%2C%20place,works%20there%2C%20so%20maybe%20that%20connection%20will%20help.%29

Reply:
From your source:
> If the parentheses occur within a sentence, place exclamation and question marks inside them, but don’t capitalize the first word in parentheses:

>> I saw Olivia (is she your cousin?) at the ice rink today

Reply:
Yes, but also: 

"When a whole sentence is in parentheses, place all punctuation inside the parentheses and capitalize the first letter: I applied for a job with Google, but I’m almost certain I’ll be rejected. (Although my aunt’s friend’s cousin works there, so maybe that connection will help.)"

So maybe we could change it to

"Often abbreviated to CSO." ?

Reply:
No, that applies only to parentheses _outside of another sentence_. If they are within a sentence, this does not apply (see above). And it looks strange...

Reply:
As per lastly applied changes the parentheses now start outside the sentence ^^